server/routes/game.controller.ts

```diff
@@ -12,10 +11,23 @@ const getAll = async (req: Request, res: Response) => {
   }
 };
 
+const getGamesById = async (req: Request, res: Response) => {
+  const id = req.query._id;
+  console.log(id);
+  try {
+    const getResult = await GameSchema.find({
+      _id: id,
+    });
+    res.status(200).send(getResult);
+  } catch (err) {
+    res.status(401).send(err);
+  }
+};
+
 //post request
 const postGame = async (req: Request, res: Response) => {
-  //console.log(GameSchema);
   let { titleOfGame, description, uId, isPrivate, gameModules } = req.body;
+  console.log("this is running");
```

Comment:
Make sure console.log are removed if no longer needed for dev. We will want to avoid pushing those to production.